platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -183,6 +191,27 @@ private ImportMigrationConfigurationAdminEntry updateNecessaryConfigurationAdmin
     if (entry.getPid().equals("org.codice.ddf.security.filter.login.Session")) {
       entry.setPid("ddf.security.http.impl.HttpSessionFactory");
     }
+    // this configuration was updated in version 2.13.6
+    // see https://github.com/codice/ddf/pull/4388
+    if (entry.getPid().equals(METACARD_VALIDITY_FILTER_PLUGIN_PID)) {
+      entry = updateMetacardValidityFilterPluginConfiguration(entry);
+    }
+    return entry;
+  }
+
+  private ImportMigrationConfigurationAdminEntry updateMetacardValidityFilterPluginConfiguration(
+      ImportMigrationConfigurationAdminEntry entry) {
+    Dictionary<String, Object> exportedProps = entry.getProperties();
+    String[] attributeMap = (String[]) exportedProps.get(ATTRIBUTE_MAP);
+
+    for (int i = 0; i < attributeMap.length; i++) {
+      String val = attributeMap[i];
+      String hostname = System.getProperty(DDF_HOSTNAME_PROP_KEY);
+      attributeMap[i] = val.replace("data-manager", hostname + "-data-manager");
```

Comment:
❗️ Technically this would turn something like "invalid-state=localhost-data-manager" into "invalid-state=localhost-localhost-data-manager". We need to handle that case. i.e. where the character preceding "data-manager" is a dash "-".